코랩 파이썬 개발 환경입니다.

# 1. 제목

## 1.1 제목

### 1.1.1 제목

- 리스트 1
- 리스트 2
- 리스트 3
 - 들여쓰기 리스트

 > 인용구문

1. 번호 리스트 1
2. 번호 리스트 2
3. 번호 리스트 3

In [4]:
x = 10 #주석.
''' 여기는
   주석입니다'''

""" 여기도
주석입니다"""


' 여기도\n주석입니다'

In [5]:
import keyword

print(keyword.kwlist)

['False', 'None', 'True', 'and', 'as', 'assert', 'async', 'await', 'break', 'class', 'continue', 'def', 'del', 'elif', 'else', 'except', 'finally', 'for', 'from', 'global', 'if', 'import', 'in', 'is', 'lambda', 'nonlocal', 'not', 'or', 'pass', 'raise', 'return', 'try', 'while', 'with', 'yield']


In [9]:
x=0b1011
a=0o52
b=0xff
print(x)
print(a)
print(b)

11
42
255


In [12]:
x = 4.23e+8
print(x)
x = 4.23e-3
print(x)

423000000.0
0.00423


In [16]:
num = 10
word = "abc"

print(str(num)+word)

10abc


In [20]:
type(True)






bool

In [26]:
x=10
y=2
z= x//y
print(z)
z=x**y
print(z)

5
100


In [32]:
a="Hello, world"
"H" in a


True

In [35]:
name = "kim"
age = 20
height=177.7
a="name %s, age %d, height %.1f."%(name,age,height)
print(a)

name kim, age 20, height 177.7.


In [37]:
decimal_num=10
binary_num = bin(decimal_num)

In [49]:
print("%d, %s" %(decimal_num,binary_num))


10, 0b1010


In [70]:
a=20
b=3
c=2
d=1
e=1.23
f=90
g=10
print("My name is Tom and I am %d years old."%(a))
print("I have %d apples, %d oranges, and %d banana."%(b,c,d))
print("The result is %.2f."%(e))
print("Your score is %d."%(f))
print("10+20=",a+g)


My name is Tom and I am 20 years old.
I have 3 apples, 2 oranges, and 1 banana.
The result is 1.23.
Your score is 90.
10+20= 30


In [76]:
A = int(input("A의 값 입력"))
B = int(input("B의 값 입력"))

SUM = A+B
DIV = A/B

print(f"{A}+{B}={SUM}")
print(f"{A}/{B}={DIV:.2f}")

A의 값 입력3
B의 값 입력2
3+2=5
3/2=1.50


![image.png]()

In [85]:
sec1 = int(input("초를 입력하세요"))
min = sec1//60
sec2 = sec1%60
print(f"{min}분 {sec2}초")

초를 입력하세요200
3분 20초


In [94]:
min1 = int(input("분을 입력하세요"))
day = min1//1440
hour = min1%1440//60
min2 = min1%1440%60
print(f"{day}일 {hour}시간 {min2}분")


분을 입력하세요1550
1일 1시간 50분


In [107]:
money = 5000000
x = money * 0.05
sum = money+x *5
print(f"원리금 합계 = {sum:.0f}")

원리금 합계 = 6250000


In [102]:
n=100
total = int(n*(n+1)/2)
print(total)

5050


In [106]:
grape=75
straw=113.5
grape_num = int(input("포도 알 개수 입력:"))
straw_num = int(input("딸기 개수 입력:"))
total = (grape*grape_num) + (straw*straw_num)
print(total)

포도 알 개수 입력:3
딸기 개수 입력:5
792.5


초를 입력하면 분과 초로 표시하는 프로그램. 예를 들어, 200초를 입력하면 3분 20초로 표현하라

분(min)을 입력 하면, 일, 시간, 분으로 출력하는 프로그램을 만들어라. (예 : 1550분은 1일 1시간 50분)

500만원을 년이율 5%로 복리 저금했을 때 5년 후의 원리금의 합계를 출력하는 프로그램

1부터 n까지의 합은 n(n+1)/2로 주어진다. 1부터 100까지의 합을 구하여 출력하는 프로그램을 작성하고 실행하라.

판매자가 딸기와 포도를 판매하고 있다. 포도 한 알의 무게는 75g이고 딸기 한 알의 무게는 113.5g이다. 사용자로부터 포도 알의 개수와 딸기의 개수를 입력 받아 총 무게를 계산하여 출력하는 프로그램을 작성하고 실행하라.
